examples/example-map-style.json

```diff
@@ -0,0 +1,7607 @@
+{
```

Comment:
Nit: This is pretty big at almost 8K lines, is that needed for the example? Could it be made a little smaller?